packages/features/sales-escrows/modals/ConfirmSalesActionModal.tsx

```diff
@@ -32,11 +31,13 @@ export const ConfirmSalesActionModal = ({
   const [isLoading, setIsLoading] = React.useState(false);
   const { enqueueSnackbar } = useSnackbar() || {};
   const { tokens } = useTokensContext();
+  const [confirmed, setConfirmed] = useState(false);
```

Comment:
I think this needs to get set to false each time the modal reopens, or the modal can not be used again without refreshing the page.

Reply:
I added a useEffect
`useEffect(() => {
    setConfirmed(false);
  }, [openConfirmation]);`